### PR TITLE
Fix errors after enabling all debug config

### PR DESCRIPTION
**Pull request by Kr159**
os: fix build error after enabling all Debug option.
 
 irqinfo was not defined in defconfig, so i include that.
 Dump_stack and Dump_all_stack was not define for armv7-a architecture. It is define now.
 wifi_set_mac_address function and wrong number of params that is rectified now.

### CODE REVIEW

File: os/arch/arm/src/armv7-a/arm_gicv2_dump.c [hunk 1]
```diff
@@ -308,4 +308,16 @@ void arm_gic_dump(const char *msg, bool all, int irq)
...
+// 		/* ToDo: implement as per the armv7-m architecture */
+// 	return;
+// }
+
+// void dump_stack(void)
+// {
+// 		/* ToDo: implement as per the armv7-m architecture */
+// 	return;
+// }
+
```
Comment (NayanaNagaraj23) [inline os/arch/arm/src/armv7-a/arm_gicv2_dump.c:322]: Remove the commented code if not necessary

File: os/board/rtl8720e/src/component/wifi/wifi_interactive_mode.c [hunk 2]
```diff
@@ -593,11 +593,11 @@ int8_t cmd_wifi_disconnect(void)
 	}
 }
 
-
 void cmd_wifi_info(int argc, char **argv)
 {
 	int i = 0;
 	u8 wlan_idx[2] = {0, 1};
+	FAR struct netdev *dev;
 	rtw_sw_statistics_t sw_stats;
```
Comment (NayanaNagaraj23) [inline os/board/rtl8720e/src/component/wifi/wifi_interactive_mode.c:601]: Isn't it dev netdev variable necessary?

**Comment by neel-samsung** [comment]
Looks like the commits in the PR are not ready to merge. please ```under dev``` or any other tag to the PR title.

**Review by neel-samsung — COMMENTED** [review]
I think this commit needs to be amended properly.

**Pull request closed without merging**